server/src/catalog.rs

```diff
@@ -97,23 +99,23 @@ pub fn new(data_folder: &str) -> Catalog {
                             &model_table_segment_group_file_schema,
                         ) {
                             model_table_metadata.push(mtd);
-                            info!("INFO: initialized model table {}", path);
+                            info!("initialized model table {}", path);
```

Comment:
@skejserjensen We should agree on the convention for writing log messages. I have been using full sentences with punctuation.

Reply:
Yeah, punctuation looks fine to me. Let's agree on that

Reply:
Great, if you have time you can just change the comments in this branch. I think it still aligns with the overall purpose.

Reply:
Fixed